scripts/changelog-ci.py

```diff
@@ -65,12 +69,18 @@ def _get_pull_request_title_and_number(event_path):
 
         return title, number
 
-    def _parse_config(self, config_file):
+    def _parse_config(self, filepath: str):
```

Comment:
We are not using annotations anywhere rights now

Reply:
Very well. I will revert this in the next commit.